text/0069-api-v2.md

```diff
@@ -64,7 +64,18 @@ GET /keyspaces
 [
   {
     name: "default",
-    id: "0"
+    id: "0",
+    metadata: {
```

Comment:
I'd like to rename it to "property" rather than "metadata". BTW, is there any guideline about what fields should be contained in this JSON object, what fields should be put outside this JSON object?

Reply:
please also update the metadata design in PD after the guideline is settled.

Reply:
> I'd like to rename it to "property" rather than "metadata".

Fair

> BTW, is there any guideline about what fields should be contained in this JSON object, what fields should be put outside this JSON object?

No. Every component can define a new config item. The component may print a warning if it meets an unrecognized property under its section (eg. TxnClient can warn the property `default-config.txn-client.abcd`).

Reply:
is it possible to put "created_at" to this field?

Reply:
If you're mentioning the `created_at` in top-level metadata, no, because it's only writable by PD. Otherwise, if you ask whether the user can set a custom `created_at` property, it's yes, the user is allowed to set whatever he wants.